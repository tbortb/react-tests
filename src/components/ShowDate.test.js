import React from 'react';
import renderer from 'react-test-renderer';
import ShowDate from './ShowDate';
describe("Snapshot Test for showdate component", () => {
    
    test('renders the same way every time', () => {
      const tree = renderer.create( <ShowDate /> ).toJSON();
      expect(tree).toMatchSnapshot();
    });
    
})