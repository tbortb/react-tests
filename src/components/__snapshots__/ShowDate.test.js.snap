// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test for showdate component renders the same way every time 1`] = `
<div>
  <span>
    The correct date is: 2017-04-05
  </span>
</div>
`;
